components/site/src/link_checking.rs

```diff
@@ -105,6 +106,10 @@ fn should_skip_by_prefix(link: &str, skip_prefixes: &[String]) -> bool {
     skip_prefixes.iter().any(|prefix| link.starts_with(prefix))
 }
 
+fn should_skip_by_file(file_path: &Path, glob_set: &GlobSet) -> bool {
```

Comment:
not sure it's worth a fn if it's only used once in this file

Reply:
should_skip_by_prefix was my point of reference here, it also is only used once in this file.

would you like me to remove both?

Reply:
No that's fine then